src/natcap/invest/ndr/ndr.py

```diff
@@ -996,6 +990,42 @@ def _mult_op(*array_nodata_list):
         gdal.GDT_Float32, target_nodata)
 
 
+def _sum_rasters(raster_path_list, target_nodata, target_result_path):
+    """Sum the rasters in `raster_path_list`.
+
+    Args:
+        raster_path_list (list): list of single band raster paths.
+        target_nodata (float): desired target nodata value.
+        target_result_path (string): path to float 32 target raster
+            multiplied where all rasters are not nodata.
```

Comment:
Do you mean __sum__ instead of __multiplied__ here?

Reply:
Oops, yeah